muikku-core-plugins/src/main/resources/META-INF/resources/scripts/gui/coursepicker.js

```diff
@@ -160,9 +165,10 @@
       window.location = CONTEXTPATH + '/workspace/' + workspaceUrl;
     }, 
     
-    _loadMore: function () {
+    _loadMore: function () {        
       this._firstResult += this._maxResults;
-      this._loadWorkspaces();
+      hasPage = true;
+      this._loadWorkspaces(hasPage);
```

Comment:
Variable hasPage isn't really needed here; this._loadWorkspaces(true); would be sufficient. Alternatively, local variables should be declared as such (var hasPage = true)